### PR TITLE
Inline svgs

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: d2l-loading-spinner.html [hunk 1]
```diff
@@ -16,64 +14,62 @@
...
-				transform-origin: left bottom;
+				width: 100%;
 			}
 
 			:host .d2l-loading-spinner-slice1,
 			:host .d2l-loading-spinner-slice2,
 			:host .d2l-loading-spinner-slice3,
 			:host .d2l-loading-spinner-slice4,
 			:host .d2l-loading-spinner-slice5 {
+				-webkit-animation-duration: 1.5s;
```
Comment (omsmith) [inline d2l-loading-spinner.html]: This just for Android?

File: d2l-loading-spinner.html [hunk 2]
```diff
@@ -16,64 +14,62 @@
...
+				-webkit-transform: rotate(270deg);
 			}
-			:host .d2l-loading-spinner-slice5{
-				-webkit-transform: rotate(344deg);
+			:host .d2l-loading-spinner-slice5 {
+				animation-name: slicespin5;
 				-webkit-animation-name: slicespin5;
 				transform: rotate(344deg);
-				animation-name: slicespin5;
+				-webkit-transform: rotate(344deg);
```
Comment (omsmith) [inline d2l-loading-spinner.html]: Wrong order, same with slice4

File: d2l-loading-spinner.html (hunk 1 above)
Comment (dlockhart) [inline d2l-loading-spinner.html]: Pretty sure the prefixes for all the animation stuff are needed for more than Android... according to [shouldiprefix](http://shouldiprefix.com/#animations) anyway...

File: d2l-loading-spinner.html [hunk 3]
```diff
@@ -149,9 +184,11 @@
 		},
 		_colorChanged: function(newColor) {
 			this.customStyle['--d2l-loading-spinner-color'] = newColor;
+			this.updateStyles();
```
Comment (dlockhart) [inline d2l-loading-spinner.html]: These 2 calls were needed for native CSS props.

File: d2l-loading-spinner.html (hunk 2 above)
Comment (dlockhart) [inline d2l-loading-spinner.html]: Good catch, thanks. Last time I try to alphabetize things...

File: d2l-loading-spinner.html (hunk 1 above)
Comment (omsmith) [inline d2l-loading-spinner.html]: I dunno, caniuse seems pretty green, http://caniuse.com/#search=animation-duration, no biggy though

Comment (dlockhart) [inline d2l-loading-spinner.html]: Cool I'll try dropping them and see what happens...

**Comment by dlockhart** [comment]
@omsmith Removed the prefixes without any impact to Chrome and Safari 10.

File: d2l-loading-spinner.html [hunk 4]
```diff
@@ -16,123 +14,121 @@
...
 			}
 
 		</style>
-		<d2l-icon class="d2l-loading-spinner-bg" icon="d2l-loading-spinner:bg"></d2l-icon>
-		<d2l-icon class="d2l-loading-spinner-slice1" icon="d2l-loading-spinner:slice"></d2l-icon>
-		<d2l-icon class="d2l-loading-spinner-slice2" icon="d2l-loading-spinner:slice"></d2l-icon>
-		<d2l-icon class="d2l-loading-spinner-slice3" icon="d2l-loading-spinner:slice"></d2l-icon>
-		<d2l-icon class="d2l-loading-spinner-slice4" icon="d2l-loading-spinner:slice"></d2l-icon>
-		<d2l-icon class="d2l-loading-spinner-slice5" icon="d2l-loading-spinner:slice"></d2l-icon>
+		<svg viewBox="0 0 42 42" class="d2l-loading-spinner-bg">
```
Comment (dbatiste) [inline d2l-loading-spinner.html]: Need `focusable="false"` on these `svg`s?  Or was that only an issue for IE10?

Comment (dlockhart) [inline d2l-loading-spinner.html]: Good call -- [IE11 was impacted](https://github.com/PolymerElements/iron-iconset-svg/issues/46) as well, so I'll add this in.

**Pull request merged**